test/lib/ufe/testComboCmd.py

```diff
@@ -890,6 +890,97 @@ def testCenterPivotUndo(self):
         assertVectorAlmostEqual(self, usdT3d.rotatePivot().vector, [0.0, 0.0, 0.0])
         assertVectorAlmostEqual(self, usdT3d.scalePivot().vector, [0.0, 0.0, 0.0])
 
+    @unittest.skipUnless(mayaUtils.ufeSupportFixLevel() > 0, "Requires center pivot Maya fix.")
```

Comment:
Use UFE fix level support to avoid running unless fix is present in Maya.

Reply:
No need to check major version?

Reply:
Nope.  UFE fix level is always increasing, and always incremented in master.